app/presenters/search_result_presenter.rb

```diff
@@ -8,29 +10,76 @@ class SearchResultPresenter
            :es_score,
            to: :search_result
 
-  def initialize(search_result, metadata)
-    @search_result = search_result
-    @metadata = metadata
+  def initialize(data = {})
+    @search_result = data[:search_result]
+    @metadata = data[:metadata]
+    @index = data[:doc_index] + 1
+    @count = data[:doc_count]
+    @finder_name = data[:finder_name]
+    @debug_score = data[:debug_score]
+    @highlight = data[:highlight]
   end
 
   def to_hash
     {
-      title: title,
-      link: link,
-      summary: summary,
-      is_historic: is_historic,
-      government_name: government_name,
-      metadata: metadata,
-      show_metadata: show_metadata,
-      format: format,
-      es_score: es_score,
+      link: {
+        text: title,
+        path: link,
+        description: summary_text,
+        data_attributes: {
+          track_category: "navFinderLinkClicked",
+          track_action: "#{@finder_name}.#{@index}",
+          track_label: link,
+          track_options: {
+            dimension28: @count,
+            dimension29: title
+          }
+        }
+      },
+      metadata: structure_metadata,
+      metadata_raw: metadata,
+      subtext: subtext,
+      highlight: @highlight,
+      highlight_text: highlight_text
     }
   end
 
   def link
     search_result.path
   end
 
+  def structure_metadata
+    return {} unless show_metadata
+
+    if show_metadata
```

Comment:
this conditional seems superfluous because you have the guard at line 52

Reply:
Have removed.